StarCellBio/settings.py

```diff
@@ -112,7 +112,6 @@
 MIDDLEWARE_CLASSES = (
     'django.middleware.common.CommonMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
```

Comment:
I've added this middleware in commit #31106d6 and found that it broke frontend tests that is why I removed it as it was in initial state.